plugins/woocommerce-blocks/tests/e2e/tests/add-to-cart-form/add-to-cart-form.block_theme.spec.ts

```diff
@@ -246,18 +275,70 @@ test.describe( `${ blockData.name } Block`, () => {
 		await admin.createNewPost();
 		await editor.insertBlock( { name: 'woocommerce/single-product' } );
 
-		await blockUtils.configureSingleProductBlock( 'Sold Individually' );
+		const productName = 'Sold Individually';
+
+		await blockUtils.configureSingleProductBlock( productName );
 		await blockUtils.enableStepperMode();
+
 		await editor.publishAndVisitPost();
 
-		const minusButton = page.locator(
-			'.wc-block-components-quantity-selector__button--minus'
+		const minusButton = page.getByLabel(
+			`Reduce quantity of ${ productName }`
 		);
-		const plusButton = page.locator(
-			'.wc-block-components-quantity-selector__button--plus'
+		const plusButton = page.getByLabel(
+			`Increase quantity of ${ productName }`
 		);
```

Comment:
There are some other instances in this file where we use the CSS class selector to locate the `minusButton` and `plusButton` (ie L246-L251). What do you think about updating those as well?

Reply:
In some cases, the Single Product is configured to pick the first product from the list rather than a fixed one. We could either:

- Add logic to find the "selected product."
- Always use the same product.

While I don't have a strong preference, keeping a "random" selection might help increase coverage. At the same time, adding logic to find the selected product feels like overkill.


What do you think?

Reply:
Good point! You mean it's an issue because we wouldn't know `productName`, right? `page.getByLabel()` defaults to not looking for exact matches, so something like `page.getByLabe( 'Increase quantity' )` should resolve to that button too, no? :thinking: 

In any case, it's a good point, I don't have a strong opinion anymore, so feel free to leave it as it makes more sense to you. By the way, in case we keep the selectors, is there any reason not to use the variables from L22-L33?

https://github.com/woocommerce/woocommerce/blob/3b48d14f3c31e4bab77e058c7017c294d57e0282/plugins/woocommerce-blocks/tests/e2e/tests/add-to-cart-form/add-to-cart-form.block_theme.spec.ts#L22-L33

Reply:
> page.getByLabel() defaults to not looking for exact matches, so something like page.getByLabe( 'Increase quantity' ) should resolve to that button too, no? 🤔

TIL! :) 

I addressed your feedback with f7c3d038e67d4debda0f0d142a775368806394d1! Thanks!